app/src/main/java/com/github/se/wanderpals/MainActivity.kt

```diff
@@ -317,6 +312,9 @@ class MainActivity : ComponentActivity() {
                   Trip(navigationActions, tripId, viewModel.getTripsRepository(), mapManager)
                 }
                 composable(Route.CREATE_TRIP) {
+                  NotificationAPI()
+                      .sendNotification(listOf(SessionManager.getNotificationToken()), "hello2")
```

Comment:
You could replace `hello2` with another string with a more meaningful `body` name, since the function `sendNotification` of NotificationAPI.kt needs one `body: String`.